sql-indent-left.el

```diff
@@ -0,0 +1,189 @@
+;;; sql-indent-left.el --- configuration options to indent sql -*- lexical-binding: t -*-
+;;
+;; Filename: sql-indent-left.el
+;; Description:
+;; Author: pierre.techoueyres@free.fr
+;; Maintainer: pierre.techoueyres@free.fr
+;; Copyright (C) 2017, Pierre Téchoueyres all rights reserved.
+;; Created:
+;; Version: pierre.techoueyres@free.fr
+;; Last-Updated:
+;;           By:
+;;     Update #: 0
+;; URL:
+;; Keywords: language sql indentation
+;; Compatibility:
+;;
+;; Features that might be required by this library:
+;;
+;;   None
+;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;
+;;; Commentary:
+;;
+;; Set configuration options to indent sql my way.
+;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;
+;;; Change log:
+;;
+;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;
+;; This program is free software; you can redistribute it and/or
+;; modify it under the terms of the GNU General Public License as
+;; published by the Free Software Foundation; either version 3, or
+;; (at your option) any later version.
+;;
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+;; General Public License for more details.
+;;
+;; You should have received a copy of the GNU General Public License
+;; along with this program; see the file COPYING.  If not, write to
+;; the Free Software Foundation, Inc., 51 Franklin Street, Fifth
+;; Floor, Boston, MA 02110-1301, USA.
+;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;
+;;; Code:
+
+(require 'sql-indent)
+
+(defun sqlind-adjust-operator-left (_syntax base-indentation)
```

Comment:
I think we should have a function that just indents the operators in the select column:

```sql
    select col1, col2
                    || col3 as composed_column,
              col4
           || col5 as composed_column2
```

It could than be added to `sql-indent.el`, just like the `sqlind-lone-semicolon`

Another function could be added that lines up the AND/OR/NOT operators (although, I wonder if we can do this using existing functions?)